development/create-static-server.js

```diff
@@ -20,6 +21,12 @@ const createStaticServer = (options) => {
         public: path.resolve('./node_modules'),
       });
     }
+
+    // Handle test-dapp-multichain URLs by removing the prefix
```

Comment:
Nit: Link issue so readers can understand why we're doing this
```suggestion
    // Handle test-dapp-multichain URLs by removing the prefix
    // See here for details: https://github.com/MetaMask/MetaMask-planning/issues/4145
```

Reply:
Seems like a good suggestion. Accepted.